CustomSkillForDataIngestion/QnAIntegrationCustomSkill/host.json

```diff
@@ -1,6 +1,6 @@
 {
   "version": "2.0",
-  "functionTimeout": "00:10:00",
+  "functionTimeout": "00:35:00",
```

Comment:
This means that we need to run in a Premium or AppService Plan: https://docs.microsoft.com/en-us/azure/azure-functions/functions-scale#timeout  Are we okay with this?  My understanding is that we wanted a simple and cheap solution that anyone could get started with.  At minimum, we should document this in the README.

Reply:
Since the max timeout on the qnamaker side is 30 minutes and concurrent updates are not supported we need to increase the timeout. It should be documented in the readme as well, will do in the next PR.